CalibMuon/DTCalibration/interface/DTCalibMuonSelection.h

```diff
@@ -4,15 +4,15 @@
 
 // user include files
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/Framework/interface/one/EDFilter.h"
 #include "FWCore/Utilities/interface/EDGetToken.h"
 
 #include "DataFormats/MuonReco/interface/MuonFwd.h"
 
 #include <vector>
 #include <string>
 
-class DTCalibMuonSelection : public edm::EDFilter {
+class DTCalibMuonSelection : public edm::one::EDFilter<> {
 public:
```

Comment:
This looks like it could be `stream`.

Reply:
@mmusich could you point me some document here?

Reply:
@yuanchao you might want to check out this twiki page: https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkModuleTypes#Stream_Module.
I also recommend this comment: https://github.com/cms-sw/cmssw/issues/36404#issuecomment-990034297

Reply:
By the way I didn't check all of the modules changed here, but there might be cases in which a `global::EDAnalyzer` might be preferred in case there are no internal states to update during the processing.

Reply:
@mmusich Then it would be non-trivial to me. This would need to be better done by DT expert.

Reply:
Mmm... @mmusich After a quick look through the files, they are either with TObject (TFile) accessing or non-immutable or thread-safe member data. So I'll just keep them with "one" version. Just I found one not used include clause that to be removed.

Reply:
thanks for checking @yuanchao, as I said I didn't check, I just suggested that you check ;)